bin/sim_interpreter.py

```diff
@@ -485,9 +487,17 @@ def token_utilizer(self, instruction_token, name, start_address, size_of_target,
         elif instruction_token == 44: #SUBF
             pass
         elif instruction_token == 45: #SUBR
-            pass
+           r2 = self.registers.get_register(arguments[1])
+           r1 = self.registers.get_register(arguments[0])
+           self.registers.set_register(self.registers.get_register(arguments[1]), sub_hex(r2, r1))
         elif instruction_token == 46: #TIXR
-            pass
+            int_val_of_X = hex2int(self.registers.get_register('X'))
+            int_val_of_X += 1 
+            val_X = int2hex(int_val_of_X, 16)
```

Comment:
int2hex alone doesnt pad with 0's or F's if the value is negative, will need to use fill_value()